spec/controllers/api/emoji_controller_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe Api::EmojiController, type: :controller do
+  # :FIXME APIのテストなのでrequest specが適当らしい。誰か書きなおしてください
+  describe 'Get #index', login_as: :current_user do
+    let(:current_user) { create(:user) }
+    subject { get :index, query: emoji_name, format: :json }
+    context '絞込した場合' do
+      let(:emoji_name) { 'octocat' }
+      it 'oktocatとoktocatのurlを含むJSONがかえること' do
+        is_expected.to be_success
+        json = JSON.parse(response.body)
+        emoji = Emoji.find_by_alias(emoji_name)
+        expect(json.first['value']).to eq emoji_name
+        expect(json.first['url']).to eq controller.view_context.emoji_path(emoji)
```

Comment:
__[rubocop]__
* Line is too long. [81/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)